libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/headerSection.spec.ts

```diff
@@ -0,0 +1,95 @@
+// (C) 2023 GoodData Corporation
+
+import * as Navigation from "../../tools/navigation";
+import { Widget } from "../../tools/widget";
+import { EditMode } from "../../tools/editMode";
+import { InsightsCatalog } from "../../tools/insightsCatalog";
+import { LayoutRow } from "../../tools/layoutRow";
+
+Cypress.Cookies.defaults({
+    preserve: ["GDCAuthTT", "GDCAuthSTT", "_csrfToken"],
+});
+
+Cypress.on("uncaught:exception", (error) => {
+    console.error("Uncaught exception cause", error);
+    return false;
+});
+
+Cypress.Cookies.debug(true);
+
+describe("Header section", { tags: ["pre-merge_isolated_tiger"] }, () => {
+    describe("Default language", () => {
+        beforeEach(() => {
+            cy.login();
+
+            Navigation.visit("dashboard/header");
+            new EditMode().isInEditMode();
+            new InsightsCatalog().waitForCatalogReload();
+        });
+
+        it("(SEPARATE) can update header for all sections", () => {
+            const section_01 = "  ";
+            const section_02 = "!@#$%^^&*";
+            const section_03 = "<button>click here</button>";
+
+            new LayoutRow(0).getHeader().setTitle(section_01).setDescription(section_01);
```

Comment:
if you use cy.fixture, you can avoid repeat these same behaviors 3 times

Reply:
tried use a fixture file